docs/RELEASE_POLICY.md

```diff
@@ -0,0 +1,93 @@
+  when a security issue arises in a supported release.
+* Minor release - When the `MINOR` field is increased, the new release adds
+  new features, bug fixes or both without changing existing user interface or
+  user facing functionality. Adding new APIs can be done in a `MINOR`
+  Firecracker release as long as it doesn’t change the functionality of the APIs
+  available in the previous release. Minor releases are shipped when features
+  are ready for production. Firecracker team may bundle multiple features
+  under the same release.
+* Major release -  When the `MAJOR` field is increased, the new release adds
+  new features, bug fixes or both, changing existing user interface or user
+  facing functionality possibly that may make it incompatible with previous
+  releases. A major release will also likely require a changes from other
+  components interacting with Firecracker - either API request, commands or
+  guest components. Please always refer to the release notes for further details.
+  Major releases are published whenever features or bug fixes changing existing
+  user interface or user facing functionality, are ready for production.
+
+## Release support
+
+Firecracker team will provide support only for Firecracker releases under
+https://github.com/firecracker-microvm/firecracker/releases.
+
+Firecracker will provide security patch releases when security issues are
+encountered for:
+
+* the last two Firecracker `vMAJOR.MINOR` releases for up to 1 year from
+  release date;
+* any Firecracker `vMAJOR.MINOR` release for at least 6 months from release date;
+* for each `vMAJOR`, the latest `MINOR` for 1 year since release date;
+
+#### Examples:
+
+1. Considering an example where the last firecracker releases are:
+    * v2.10.0 released on 05.2022
+    * v2.11.0 released on 07.2022
+    * v2.12.0 released on 09.2022
+
+    In case of an event occurring in 10.2022, all three releases will be patched
+    since less than 6 months elapsed from their MINOR release time.
+
+1. Considering an example where the last firecracker releases are:
+    * v2.10.0 released on 05.2022
+    * v2.11.0 released on 07.2022
+    * v2.12.0 released on 09.2022
+
+    In case of of an event occurring in 05.2023, v2.11 and v2.12 will be patched
+    since those were the last 2 firecracker major releases and less than an year
+    passed since their release time.
+
+1. Considering an example where the last firecracker releases are:
+    * v2.14.0 released on 05.2022
+    * v3.0.0 released on 07.2022
+    * v3.1.0 released on 09.2022
+
+    In case of of an event occurring in 01.2023, v2.14 will be patched since is
+    the last minor of v2 and has less than one year since release while v3.0 and
+    v3.1 will be patched since were the last two Firecracker releases and
+    less than 6 months have passed since release time.
+
+
+## Developer preview features
+
+A feature is released as developer preview if it’s marked as such in the
```

Comment:
I think "released" as developer preview is confusing since we also use the word release for non-dev-preview features. Maybe just say A feature is "in" developer preview / Dev Preview.